build.sbt

```diff
@@ -16,6 +16,7 @@ inThisBuild(
   List(
     name               := "zio-logging",
     ciEnabledBranches  := Seq("master"),
+    // ciJvmOptions ++= Seq("-Xmx6g", "-Xss2m", "-XX:+UseG1GC"),
```

Comment:
is this some leftover ?

Reply:
is was rendered in the ci.yaml but not properly picked up in the job. If you check the jobs here then you will see it was picked up but later in the job it still seemed to have only the default 1GB of memory: https://github.com/zio/zio-logging/actions/runs/11993647760/job/33434815940
So for now I added a `.jvmopts` file to fix it.